shared/common-adapters/scroll-view.desktop.tsx

```diff
@@ -46,13 +46,18 @@ const ScrollView = React.forwardRef(function ScrollView(props: Props, ref) {
     }),
     [divRef]
   )
+
+  React.useEffect(() => {
+    const options = {passive: true}
```

Comment:
add passive for some perf gains, there isn't a native way to do this in react annoyingly